docker/prod_vpp_agent/Dockerfile

```diff
@@ -2,12 +2,15 @@ FROM ubuntu:16.04
 
 MAINTAINER jozef.mak@pantheon.tech
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update \
+ && apt-get install -y \
         # general tools
         iproute2 iputils-ping inetutils-traceroute \
         # vpp requirements
         openssl python libapr1 libnuma1 \
-        supervisor
+        supervisor \
+ && apt-get clean \
```

Comment:
Running `apt-get clean` is unnecessary, since the official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.

Reply:
@ondrej-fabry: I've removed that line from the Dockerfile.

PTAL